<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1265</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>1261</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #C0E5F0 </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>1271</width>
      <height>571</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>196</x>
        <y>129</y>
        <width>901</width>
        <height>381</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="styleSheet">
        <string notr="true">#lineEdit,#lineEdit_nom,#lineEdit_prenom,#lineEdit_adresse,#lineEdit_tel,#lineEdit_7,#date_naissance{
background-color :  #69BCD4 ; 



   border: 0.25px solid black;
    border-radius: 10px;
    color: black; 
    font-size: 15px;
}

#ajouter,#pushButton_9{
background-color :  #F1B655; 



   border: 0.5px solid black;
    border-radius: 15px;
    color: solid black; 
    font-size: 15px;
}</string>
       </property>
       <attribute name="title">
        <string>Ajouter Personnel</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_adresse">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>200</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>50</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>60</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>270</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>210</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Adresse</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ajouter">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>210</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>260</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Tel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Date de naissance</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nom et Pr√©nom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_tel">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>250</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>100</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="date_naissance">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>150</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>170</y>
          <width>141</width>
          <height>141</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <property name="styleSheet">
        <string notr="true">#pdf,#recherche,#stat{
background-color :  #F1B655; 



   border: 0.5px solid black;
    border-radius: 15px;
    color: solid black; 
    font-size: 15px;
}</string>
       </property>
       <attribute name="title">
        <string>Liste du Personnel</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>771</width>
          <height>241</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="recherche">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>290</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Recherche</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pdf">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>290</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
       <widget class="QPushButton" name="stat">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>290</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Statistiques</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <property name="styleSheet">
        <string notr="true">#lineEdit_2,#lineEdit_nom_2,#lineEdit_prenom_2,#lineEdit_adresse_2,#lineEdit_tel_2,#lineEdit_7,#date_naissance_2{
background-color :  #69BCD4 ; 



   border: 0.25px solid black;
    border-radius: 10px;
    color: black; 
    font-size: 15px;
}

#modifier,#pushButton_10{
background-color :  #F1B655; 



   border: 0.5px solid black;
    border-radius: 15px;
    color: solid black; 
    font-size: 15px;
}</string>
       </property>
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>50</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>40</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>250</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Tel</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom_2">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>90</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="modifier">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>290</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="date_naissance_2">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>140</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_tel_2">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>240</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string> Nom et Pr√©nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Date de naissance</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_adresse_2">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>190</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>200</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Adresse</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_10">
        <property name="geometry">
         <rect>
          <x>720</x>
          <y>290</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>435</x>
          <y>30</y>
          <width>441</width>
          <height>241</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <property name="styleSheet">
        <string notr="true">#supprimer,#pushButton_20{
background-color :  #F1B655; 



   border: 0.5px solid black;
    border-radius: 15px;
    color: solid black; 
    font-size: 15px;
}</string>
       </property>
       <attribute name="title">
        <string>Supprimer </string>
       </attribute>
       <widget class="QLineEdit" name="supp_id">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>100</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color :  #69BCD4 ; 



   border: 0.25px solid black;
    border-radius: 10px;
    color: black; 
    font-size: 15px;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="supprimer">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>270</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_20">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>270</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ID du personnel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>40</y>
          <width>141</width>
          <height>131</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>141</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1265</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
